### PR TITLE
Add Intl.Collator.prototype[@@toStringTag]

### PR DESCRIPTION
**Pull request by pmarkee**
Related issue: #251

Signed-off-by: Peter Marki marpeter@inf.u-szeged.hu

### CODE REVIEW

**Comment by pmarkee** [comment]
@clover2123 the only thing I found that was missing from `Intl.Collator` based on the standard was this `Intl.Collator.prototype[@@toStringTag]` property, but I didn't check very in-depth. Do you know about anything else that is missing/incomplete in `Intl.Collator`? If there is, I can add that to this patch.

**Comment by pmarkee** [comment]
Depends on #260 for testing.

File: src/runtime/GlobalObjectBuiltinIntl.cpp [hunk 1]
```diff
@@ -2560,6 +2561,9 @@ void GlobalObject::installIntl(ExecutionState& state)
     m_intlCollator->getFunctionPrototype(state).asObject()->defineOwnProperty(state, state.context()->staticStrings().resolvedOptions,
                                                                               ObjectPropertyDescriptor(new FunctionObject(state, NativeFunctionInfo(strings->resolvedOptions, builtinIntlCollatorResolvedOptions, 0, nullptr, NativeFunctionInfo::Strict)), (ObjectPropertyDescriptor::PresentAttribute)(ObjectPropertyDescriptor::ConfigurablePresent | ObjectPropertyDescriptor::WritablePresent)));
 
+    m_intlCollator->getFunctionPrototype(state).asObject()->defineOwnProperty(state, ObjectPropertyName(state, state.context()->vmInstance()->globalSymbols().toStringTag),
+                                                                              ObjectPropertyDescriptor(strings->Object.string(), (ObjectPropertyDescriptor::PresentAttribute)(ObjectPropertyDescriptor::NotPresent)));
```
Comment (clover2123) [inline src/runtime/GlobalObjectBuiltinIntl.cpp]: Could you check the spec? 
`Intl.Collator.prototype [ @@toStringTag ]` has configurable attribute according to the following spec.
> This property has the attributes { [[Writable]]: false, [[Enumerable]]: false, [[Configurable]]: true }.

Comment (pmarkee) [reply src/runtime/GlobalObjectBuiltinIntl.cpp]: True, I misread the spec, thanks.

**Pull request merged**